committee/README.md

```diff
@@ -0,0 +1,20 @@
+# PMC
+
+PMC (Project Management Committee) is to manage and oversee the entire TiKV community, also defines the project values and structure. The PMC consists of Core Maintainers and Organizer Committee Chair.
+
+## Members
```

Comment:
This list is missing some of our maintainers, despite it saying it includes our maintainers.